guide/getting-started/developers.md

```diff
@@ -52,9 +52,9 @@ Add an entry to the `module.rules` array to load CSS. You may also provide other
 }
 ```
 
-### 2. Import the TDS styles
+### 2. Import the TDS css-reset styles
 
-Import the TDS stylesheet into the main entry point of your application. It contains a small amount of page level styles and the pre-compiled components' styles. We use CSS Modules to generate unique class names, which guarantees that TDS component styles will not conflict with your application's class names.
+Import the TDS stylesheet into the main entry point of your application. It contains a small amount of page level styles.
```

Comment:
Same comment as below with the Colours section. Now that the CSS reset is a separate package, we should tell people to install it!

Reply:
We have the last section talking about colours
`### 4. Use TDS colours`

I will add a note to install the dependency.

Reply:
And I added a blurb to add the css-reset dependency in 
`### 2. Import the TDS css-reset styles`